src/engine/client/backend/opengl/backend_opengl3.cpp

```diff
@@ -30,7 +30,7 @@ static constexpr GLenum BUFFER_INIT_VERTEX_TARGET = GL_COPY_WRITE_BUFFER;
 #endif
 
 // ------------ CCommandProcessorFragment_OpenGL3_3
-void CCommandProcessorFragment_OpenGL3_3::UseProgram(CGLSLTWProgram *pProgram)
+void CCommandProcessorFragmentOpenGL33::UseProgram(CGLSLTWProgram *pProgram)
```

Comment:
```suggestion
void CCommandProcessorFragmentOpenGl3_3::UseProgram(CGLSLTWProgram *pProgram)
```
I think it makes sense to separate the 3 from the three. Comment above also needs to be modified.